common/persistence/cassandra/cassandraTaskPersistence.go

```diff
@@ -24,174 +24,68 @@ package cassandra
 import (
```

Comment:
Should update this file name?

Reply:
I will update them later when we move the files to nosql package.